include/common/test.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef TEST_H
+#define TEST_H
+#ifdef TEST
```

Comment:
Please put blank lines around the `#ifdef TEST` and its respective `#endif` line

Reply:
done